client/src/styles/userforms-cms.scss

```diff
@@ -166,3 +166,26 @@
 .cms-preview {
   z-index: 2;
 }
+
+// Custom styles for userforms fields in the CMS
+.userform-field {
+  &__allowed-length {
+    .form__fieldgroup-item {
+      // Removing default margin-top from field group, this one already has one in its parent container
+      margin-top: 0;
+    }
+
+    .field.text {
+      min-width: auto;
+    }
+
+    input.text {
+      width: 4rem;
```

Comment:
Did you check for appropriate variables to use here?

Reply:
Yeah, this is kind of specific so it evaluates as four characters wide. I could use a spacer and multiply it perhaps?

Reply:
Na it's fine. Just double checking.